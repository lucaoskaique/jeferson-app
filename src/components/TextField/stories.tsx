import { Story, Meta } from '@storybook/react'

import TextField from '.'

export default {
  title: 'TextField',
  component: TextField
} as Meta

export const Default: Story = () => <TextField />
