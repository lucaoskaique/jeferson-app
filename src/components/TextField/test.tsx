import { renderWithTheme } from 'utils/tests/helpers'

import TextField from '.'

describe('<TextField />', () => {
  it('should render the heading', () => {
    const { container } = renderWithTheme(<TextField />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
