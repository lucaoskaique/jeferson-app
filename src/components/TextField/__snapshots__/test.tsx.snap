// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TextField /> should render the heading 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background: #EAEAEA;
  border-radius: 1rem;
  padding: 0 1.6rem;
  border: 0.2rem solid;
  border-color: #EAEAEA;
}

.c0:focus-within {
  box-shadow: 0 0 0.5rem #EC6330;
}

.c1 {
  color: #030517;
  font-family: Poppins,-apple-system,BlinkMacSystemFont,'Segoe UI',Roboto,Oxygen,Ubuntu,Cantarell,'Open Sans','Helvetica Neue',sans-serif;
  font-size: 1.6rem;
  padding: 0.8rem 0;
  padding-left: 1.6rem;
  background: transparent;
  border: 0;
  outline: none;
  width: 100%;
}

.c1:-webkit-autofill {
  -webkit-box-shadow: 0 0 0 2.4rem #EAEAEA inset;
  -webkit-filter: none;
  filter: none;
}

.c1:-webkit-autofill::first-line {
  font-family: Poppins,-apple-system,BlinkMacSystemFont,'Segoe UI',Roboto,Oxygen,Ubuntu,Cantarell,'Open Sans','Helvetica Neue',sans-serif;
  font-size: 1.6rem;
}

<div
  class=""
>
  <div
    class="c0"
  >
    <input
      class="c1"
      type="text"
      value=""
    />
  </div>
</div>
`
